opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -81,6 +97,54 @@ namespace Opm {
                 for (size_t globalIndex = 0; globalIndex < multpvData.size(); globalIndex++)
                     values[globalIndex] *= multpvData[globalIndex];
             }
+
+            // deal with the region multiplier for porosity
+            if (deck->hasKeyword("MULTREGP")) {
+                const DeckKeyword& multregpKeyword = deck->getKeyword("MULTREGP");
+                for (int recordIdx = 0; recordIdx < multregpKeyword.size(); ++recordIdx) {
+                    const DeckRecord& multregpRecord = multregpKeyword.getRecord(recordIdx);
+
+                    int regionIdx = multregpRecord.getItem("REGION").template get<int>(0);
+                    std::string regionType = multregpRecord.getItem("REGION_TYPE").template get<std::string>(0);
+                    double multValue = multregpRecord.getItem("MULTIPLIER").template get<double>(0);
```

Comment:
that's because in some instances the compiler can't figure out if this is meant to be a call to a template method or `get` is a public attribute, i.e. whether the call should be interpreted as

```c++
((multregpRecord.get < T) >)()
```

(which would be a syntax error, but I believe there are corner cases where it would result in valid code.) this happens if the type of the `multregpObject` object is a template argument.

> Shouldn't be needed here though.

I prefer to leave it like it is because it will work even if the type of the `multregpRecord` is made a template...